config.inc.php.dist

```diff
@@ -6,12 +6,12 @@ define('USER', 'guest');
 define('PASS', 'guest');
 define('VHOST', '');
 
-define('MEDIA_NAME', '');
+define('MEDIA_NAME', 'mtt');
 define('MEDIA_STORAGE_PATH', '');
```

Comment:
I think it's better to add '/tmp/files/' here.
If you would like to add default configuration, I prefer to add for all parameters. That will be clear and not ambiguous.

What do you think ?